packages/pages-components/src/map/coordinate.ts

```diff
@@ -130,16 +132,21 @@ class Coordinate {
    * and one one {@link LONGITUDE_ALIASES | longitude alias}.
    * @param longitude - Optional only if the first argument is a {@link Coordinate}-like object
    */
-  constructor(latitudeOrObject: number | object, longitude?: number) {
-    const latitude = latitudeOrObject;
+  constructor(latitudeOrObject: number | object, long?: number) {
+    let latitude = latitudeOrObject;
+    let longitude: number | latLngFunction | undefined = long;
 
     this._lat = 0;
     this._lon = 0;
 
-    if (typeof latitudeOrObject === "object") {
-      this._lat = findValue(latitudeOrObject, LATITUDE_ALIASES) ?? 0;
-      this._lon = findValue(latitudeOrObject, LONGITUDE_ALIASES) ?? 0;
-    } else if (typeof latitude === "number" && longitude) {
+    if (typeof latitudeOrObject == "object") {
```

Comment:
Sorry I'm not sure what you mean when you ask whether long should be considered?

Reply:
from what I understand the cases here are - 
1) latitudeOrObject is a coordinate or something similar of the form like `{lat: 123, long: 123}` and longitude is undefined, so findValue would return numbers for both
2) latitudeOrObject is an object from a map provider like this one from Google - https://developers.google.com/maps/documentation/javascript/reference/coordinates#LatLng and longitude is undefined, where findValue() would return the `lat()` function for the latitude and the `lng()` function for the longitude, which would be called to return numbers
3) latitudeOrObject and longitude are both numbers - in this case the `if (typeof latitude == "number" && typeof longitude == "number")` was to make typescript happy 
does this help or is that not what you were asking about?

Reply:
I was mostly getting at the fact that you changed it from if/else to if/if. If latitudeOrObject is an object then why do you need to also check
`if (typeof latitude == "number" && typeof longitude == "number")`

Reply:
That was only because typescript wasn't sure that it was a number unless I put it in an if statement for typeof == number kind of like what you had suggested here https://github.com/yext/js/pull/62#discussion_r1824690714

Reply:
Shouldn't the second if be an else if though? If you hit the first if condition latitude/longitude are already set. It's confusing to make a bunch of ifs when we're really checking the types of the user input parms. If we hit the first case the second case shouldn't be necessary.

Reply:
Oh yeah I see what you're saying ty!